src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -15,7 +15,6 @@ var DOMProperty = require('DOMProperty');
 
 var MUST_USE_PROPERTY = DOMProperty.injection.MUST_USE_PROPERTY;
 var HAS_BOOLEAN_VALUE = DOMProperty.injection.HAS_BOOLEAN_VALUE;
-var HAS_SIDE_EFFECTS = DOMProperty.injection.HAS_SIDE_EFFECTS;
```

Comment:
Yeah, I will follow up once this has merged.  But I don't want to make this diff any more complicated (it's taking long enough for people to review already), and git seems to make my life pretty miserable if I try to stack diffs and then changes are requested to the base diff, so I'll wait until after this has merged.